apps/ios/SimpleXChat/AppGroup.swift

```diff
@@ -276,6 +276,68 @@ public class IntDefault: Default<Int> {
     }
 }
 
+public class StringDefault: Default<String> {
```

Comment:
don't think we need it - and not in app group anyway - it's a usual string default.